StRoot/StETofUtil/StETofGeometry.cxx

```diff
@@ -578,18 +643,18 @@ void
 StETofGeometry::init( TGeoManager* geoManager )
 {
     double safetyMargins[ 2 ] = { 0., 0. };
-    init( geoManager, safetyMargins, false );
+    init( NULL, geoManager, safetyMargins, false );
 }
 
 void
 StETofGeometry::init( TGeoManager* geoManager, const double* safetyMargins )
 {
-    init( geoManager, safetyMargins, false );
+    init( NULL, geoManager, safetyMargins, false );
```

Comment:
In C++ we use `nullptr`

Reply:
fixed in latest commit

Reply:
Removed with the removal of the StMaker-Pointer to load database